### ISSUE TITLE
can not show bar3D

### ISSUE THREAD

**Issue by Jackhsu12x** [go-echarts/go-echarts#312]
Uncaught TypeError: Cannot read properties of undefined (reading 'util')
    at Object.<anonymous> (echarts-gl.min.js:1:1727)
    at t (echarts-gl.min.js:1:385)
    at Object.<anonymous> (echarts-gl.min.js:1:272294)
    at t (echarts-gl.min.js:1:385)
    at Object.<anonymous> (echarts-gl.min.js:1:268824)
    at t (echarts-gl.min.js:1:385)
    at Object.<anonymous> (echarts-gl.min.js:1:268417)
    at t (echarts-gl.min.js:1:385)
    at echarts-gl.min.js:1:742
    at echarts-gl.min.js:1:753
(anonymous) @ echarts-gl.min.js:1
t @ echarts-gl.min.js:1
(anonymous) @ echarts-gl.min.js:1
t @ echarts-gl.min.js:1
(anonymous) @ echarts-gl.min.js:1
t @ echarts-gl.min.js:1
(anonymous) @ echarts-gl.min.js:1
t @ echarts-gl.min.js:1
(anonymous) @ echarts-gl.min.js:1
(anonymous) @ echarts-gl.min.js:1
(anonymous) @ echarts-gl.min.js:1
(anonymous) @ echarts-gl.min.js:1
bar3d.html:27 Uncaught ReferenceError: action_fBRGAQrmHEJT is not defined
    at bar3d.html:27:41
(anonymous) @ bar3d.html:27
bar3d.html:41 Uncaught ReferenceError: action_VRpSiwRaOMqC is not defined
    at bar3d.html:41:41
(anonymous) @ bar3d.html:41
bar3d.html:55 Uncaught ReferenceError: action_qNYmfhfvnTEQ is not defined
    at bar3d.html:55:41
(anonymous) @ bar3d.html:55
bar3d.html:69 Uncaught ReferenceError: action_UlefdEWkOTAD is not defined
    at bar3d.html:69:41

**Comment by Koooooo-7** [comment]
Hi @Jackhsu12x , As per to the `echarts-gl.min.js` doesn't support the newer echarts version which we upgrade to `v5.4.3`.
The workaround is downgrade the `echarts` version to 4.3.
Sorry for the inconvenient, we will do the compatibility asap.
- On page usage
```go
        page := components.NewPage()
	page.AssetsHost = "https://cdn.jsdelivr.net/npm/echarts@4.3/dist/"
```

- On single chart usage
```go
	bar := charts.NewBar()
	bar.SetGlobalOptions(
		charts.WithInitializationOpts(opts.Initialization{
			AssetsHost: "https://cdn.jsdelivr.net/npm/echarts@4.3/dist/",
		}),
        )

```

**Comment by Koooooo-7** [comment]
solved in `v2.3.x`.
